### DISCUSSION TITLE
Windows updates via web interface

### DISCUSSION THREAD

**Discussion by Thecrafterja** [Ylianst/MeshCentral#4401] [Q&A] [answered]
I know, this question has been asked a hundred times.

Is there any possibility to perform Windows updates on the clients via the web interface?

**Comment by si458** [comment, marked as answer]
NO
for something like this you need a remote monitoring and management software like tacticalrmm
https://github.com/amidaware/tacticalrmm

**Comment by PathfinderNetworks** [comment]
I'd LOVE to see features like this added to MeshCentral.  Definitely something that has been on my wish list since MeshCentral 1 back in the day.

**Comment by silversword411** [comment]
PSWindowsUpdate script is the only other answer because windows patching is #NightmareInducing
